static/app/views/dashboardsV2/widgetCard/index.tsx

```diff
@@ -94,6 +101,9 @@ class WidgetCard extends React.Component<Props> {
           <IconClick data-test-id="widget-edit" onClick={onEdit}>
             <IconEdit color="textColor" />
           </IconClick>
+          <IconClick data-test-id="widget-duplicate" onClick={onDuplicate}>
```

Comment:
instead of a `data-test-id` attribute, should we use `aria-label` here?

Reply:
👍 updated to aria-label!